server/tables/tables.go

```diff
@@ -114,9 +114,9 @@ type Invocation struct {
 	DurationUsec                   int64
 	UploadThroughputBytesPerSecond int64
 	ActionCount                    int64
-	Perms                          int `gorm:"index:perms;type:int(11);default:NULL"`
+	Perms                          int `gorm:"index:perms;type:int;default:NULL"`
```

Comment:
Can you check if 'int32' maps to the correct type in postgres? 
I think we're better off using a more specific type int32 that maps to 'int' in mysql vs having to manually override the type via tags.

Reply:
Looks like `int32` maps to `int` in postgres as well, so sgtm